modules/ROOT/nav.adoc

```diff
@@ -3,6 +3,7 @@
 * xref:design-modify-raml-specs-conform.adoc[Modify Published RAML API Specifications to Conform Completely to RAML 0.8 or 1.0]
  ** xref:design-common-problems-raml-08.adoc[Common Problems in Conforming to RAML 0.8]
  ** xref:design-common-problems-raml-10.adoc[Common Problems in Conforming to RAML 1.0]
+  *** xref:design-named-examples.adoc[Guide to Named Examples]
```

Comment:
@kbostrom , yes, I agree that the placement of this topic doesn't conform to the guideline you mention. However, I can't move this material into the container topic because that topic is already very long; it contains a large number of sections that each explain how to resolve a particular error message. Not all of those error messages have to do with named examples. Moreover, I can't bump this topic up one level with its placement implying that it is relevant to both RAML 0.8 and RAML 1.0. It is relevant only to RAML 1.0. Well, I guess I could bump the topic up and change the title to "Guide to Named Examples in RAML 1.0". Hey, there's an idea. I'll do that.